src/test/java/UploadReceiptServletTest.java

```diff
@@ -312,17 +344,19 @@ public void doPostRemovesDuplicateCategories() throws IOException, ReceiptAnalys
   public void doPostSanitizesCategories() throws IOException, ReceiptAnalysisException {
```

Comment:
nitnit: maybe update the test name to say that it tests that categories are sanitized in addition to returning both user and generated categories?